File: Основи програмування/Курс 1/Lab1.java
public class Lab1 {
public static void main (String args[]){
double A=1.63,B=15.65,C=8.21,D=1.265,E=2.035,x=0.65,k,e=2.7,W;
k=B/(B+C);
W=(k-Math.pow(e,(-A*x)))/x*(1+D*x)*(1+E*x);
System.out.println("k= "+k);
System.out.println("W= "+W);
}
}

File: Основи програмування/Курс 2/Lab1.cpp
#include <iostream>
#include <math>
void main(){
  const float π=3.1415926; /*Ініціалізуємо π*/
  float x=2, y;
  y=pow(x*x+7.2,1/5)-fabs(x/5)+sin(π*x/3);
  cout<<"\n y="<<y<<"/n";
}
